### PR TITLE
Improve code fixing prompt

### PR DESCRIPTION
**Pull request by cjx10**
Slight improvement on coverage diff on multiple projects
#44 
Also tries to address #126

### CODE REVIEW

**Comment by cjx10** [comment]
JOB: https://pantheon.corp.google.com/kubernetes/job/us-central1-c/llm-experiment/default/ofg-pr-152-jim
REPORT: https://llm-exp.oss-fuzz.com/Result-reports/ofg-pr/2024-03-12-152-jim-comparison/index.html
BUCKET: https://pantheon.corp.google.com/storage/browser/oss-fuzz-gcb-experiment-run-logs/Result-reports/ofg-pr/2024-03-12-152-jim-comparison
BUCKET GS: `gs://oss-fuzz-gcb-experiment-run-logs/Result-reports/ofg-pr/2024-03-12-152-jim-comparison`

**Comment by DonggeLiu** [comment]
Is the result better than the previous ones?

File: prompts/template_xml/fixer_priming.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Given the following C++ program and its build error message, fix the code to achieve the original functionality.
+Given the following C++ fuzz harness and its build error message, fix the code to make it build for fuzzing.
 
-If a function is missing, fix it by including the related libraries.
+If there is undeclared identifier or unknown type name error, fix it by finding and including the related libraries.
 
-If the error contains "undefined reference to", this means one or more of the header <code>#include</code> directives need to be wrapped with <code>extern "C" {}</code> for the object linking to succeed. You must fix this by rewriting the relevant <code>#include</code> and wrapping it with <code>extern "C" {}</code>.
+Note that some codes may need to be wrapped with <code>extern "C"</code> as their source is C program.
```
Comment (mihaimaruseac) [inline prompts/template_xml/fixer_priming.txt:5]:
```suggestion
Note that some code may need to be wrapped with <code>extern "C"</code> as their source is C program.
```

**Review by mihaimaruseac — APPROVED** [review]

File: prompts/template_xml/fixer_priming.txt (hunk 1 above)
Comment (cjx10) [reply prompts/template_xml/fixer_priming.txt]: Thanks Mihai 😃

**Pull request merged**